infra/terraform/modules/control_panel_api/redis.tf

```diff
@@ -0,0 +1,40 @@
+resource "aws_elasticache_replication_group" "control_panel_redis" {
+  replication_group_id          = "${terraform.workspace}-control-panel-redis"
+  replication_group_description = "${terraform.workspace} CP Redis cluster"
+
+  engine         = "redis"
+  engine_version = "5.0.6"
```

Comment:
might be worth making this a variable so we can more easily upgrade at later time?

Reply:
Cool, I'll move to a variable and default to `5.0.6`. In theory AWS will auto-upgrade the minor versions anyway which I guess will confuse Terraform a little but that's always the case with DB versions I think...

Anyway, thanks for the review @andylightfoot ❤️  - I'll make this change and merge/apply in `alpha` so at least the Redis cluster will be ready and I can get the endpoint and put in the config 👌

Reply:
@andylightfoot FYI: [added `redis_engine_version` variable](https://github.com/ministryofjustice/analytics-platform-ops/pull/368/files#diff-0c4b2b1e7fb9bdf2271e8ea91829d9faR47-R49) and applying to `alpha` now...It's a bit slow at spawning a new Redis cluster but I don't imagine it failing but I'll merge after that.